features/audit.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Display audit entities list.
+
+  Scenario: Audit show should have following properties
+    Then I run the code:
+    """
+    $conjur.audit_show
+    """
+
+    Then the JSON should have the following:
```

Comment:
Is there any documentation for what these fields represent? (what is sdata?)

Reply:
Don't think we have a documentation for audit database table columns, but it will be great if we will add it to https://docs.conjur.org/Latest/en/Content/API/ !

Reply:
Can you please create a story to capture this work?

Reply:
sure!